datasets/superb/README.md

```diff
@@ -87,6 +87,34 @@ Automatic Speech Recognition (ASR) transcribes utterances into words. While PR a
 
 Keyword Spotting (KS) detects preregistered keywords by classifying utterances into a predefined set of words. The task is usually performed on-device for the fast response time. Thus, accuracy, model size, and inference time are all crucial. SUPERB uses the widely used [Speech Commands dataset v1.0](https://www.tensorflow.org/datasets/catalog/speech_commands) for the task. The dataset consists of ten classes of keywords, a class for silence, and an unknown class to include the false positive. The evaluation metric is accuracy (ACC)
 
+##### Example of usage:
```

Comment:
Would it be too much code to show how `map_to_array` and `sample_noise` should be used in a simple demo? What I have in mind is something like:

```python
from datasets import load_dataset

superb_ks = load_dataset("superb", "ks", split="train")
# Create audio data array
superb_ks = superb_ks.map(map_to_array)
# What do I do with sample_noise?
```

No strong opinion on this, just an idea :)

Reply:
I'm in favor of this too, but there was [this discussion](https://github.com/huggingface/datasets/pull/2661#discussion_r682712915) with @albertvillanova  about a similar issue with SD, but I don't have enough context. Maybe you guys can elaborate why it was like that for SD?

Reply:
Ah thanks for reminding me! 

In the SD case, we weren't sure if the approach in `generate_label` was 100% consistent with the evaluation in `s3prl` and we wanted to test everything end-to-end before publicising this to the end-user.

 For `ks` do you happen to know if using your `sample_noise` function leads to consisten results with `s3prl`? If yes, it might be helpful to show some pseudo-code that helps a casual end-user get started.

Again, no string opinion on this so happy to leave things as-is and revisit it once the whole benchmark is up

Reply:
The sampling is theoretically consistent with s3prl's dataset, but to be 100% sure, we'd have to fix random seeds on both sides. For now I added links to the S3PRL and TFDS examples for this dataset for future reference.